custom/enikshay/management/commands/nikshay_registration_notification_time_report.py

```diff
@@ -76,17 +79,14 @@ def handle(self, days, *args, **options):
                 })
 
             if email_to:
-                attachment = {
-                    'title': result_file_name,
-                    'mimetype': 'text/csv',
-                    'file_obj': csvfile,
-                }
-
-                message = "Report for time taken for registration notifications yesterday till now"
-                send_html_email_async(
+                email_to = list(email_to) if not isinstance(email_to, basestring) else [email_to]
+                csvfile = open(result_file_name)
+                email = EmailMessage(
```

Comment:
curious what prompted this change?

Reply:
the issue i was trying to address was `send_html_email_async` expects a StringIO object on which it would get_value later. Here we had a csv file so i thought it was better to just attach the file here then.